pkgs/applications/science/programming/scyther/cli.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, cmake, glibc, makeWrapper, fetchFromGitHub, flex, bison, source }:
+stdenv.mkDerivation { 
+  name = "scyther-cli-${source.rev}";
+  src = source;
+  preConfigure = ''
+    cd src
+    # Since we're not in a git dir, the normal command this project uses to create this file wouldn't work
+    printf "#define TAGVERSION \"${source.rev}\"\n" >> version.h
+  '';
+
+  dontUseCmakeBuildDir = true;
+  cmakeFlags = [ "-DCMAKE_C_FLAGS=\"-std=gnu89\"" ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp scyther-linux $out/bin
+    ln -s $out/bin/scyther-linux $out/bin/scyther-cli
+  '';
+  
+  buildInputs = [
+    glibc.static
+    cmake
+    flex
+    bison
+  ];
+}
```

Comment:
Thanks for the review!

Since only the drv in default.nix is intended to be used I only put meta there, it would be the same for all (internal) drvs.